src/quick-lint-js/error_decorator.h

```diff
@@ -0,0 +1,54 @@
+// quick-lint-js finds bugs in JavaScript programs.
```

Comment:
Nit: Call this file `error-decorator.h` instead of `error_decorator.h`

Reply:
I changed the name of the added files and their new names follow the naming convention;